.github/workflows/test_client.yaml

```diff
@@ -34,7 +34,10 @@ jobs:
         run: |
           cd client/hwlib/
           ./debian/vendor-rust.sh
-          echo -e '[source.crates-io]\nreplace-with = "vendored-sources"\n\n[source.vendored-sources]\ndirectory = "'"$PWD"'/vendor"' > ~/.cargo/config.toml
+          echo '[source.crates-io]' >>  ~/.cargo/config.toml
```

Comment:
This looks like a here-doc
cat >>~/.cargo/config.toml << __CONFIG__
[source.creates-io]
replace-with = "vendored-sources"
[source.vendored-sources]
directory = "$PWD/vendor"
__CONFIG__